Document.cs

```diff
@@ -74,36 +72,68 @@ public class Document : IDisposable
 
 		#region Properties
 
-		DateTime? _ReleaseDate;
-		static readonly object releaseDateLock = new object();
+		private int _majorVersion;
+		private int _minorVersion;
+		private int _pointVersion;
+
 		/// <summary>
-		/// Gets the release date of the underlying Tidy library.
+		/// Returns the major version of libtidy.dll
 		/// </summary>
-		public DateTime ReleaseDate
+		public int MajorVersion
```

Comment:
I don't see anything new that uses these version numbers...do we need them? (I think you might have taken out the users of this following another suggestion I made)